e2e/steps_file.js

```diff
@@ -85,34 +102,39 @@ module.exports = function () {
       this.wait(2); // in seconds
```

Comment:
Maybe we should wait for login page to appear instead of two secs?

Reply:
Will try it and comment finding back.

Reply:
After adding regular wait for content tests started failing on:

```
Sign in as local authority:
     Text "Sign in" was not found on page after 7.5 sec
Evaluation failed: TypeError: Cannot read property 'innerText' of null
```

Not sure, screenshots show login page correctly. Anyway that wait of 2 seconds seemed to be fairly robust ;)